script/brew-publish

```diff
@@ -1,4 +1,4 @@
-#!/bin/bash
```

Comment:
bash is installed in `/usr/local/bin/bash`.
It's an optional package and not included in FreeBSD base system.